geodiff/src/drivers/postgresdriver.h

```diff
@@ -0,0 +1,33 @@
+/*
+ GEODIFF - MIT License
+ Copyright (C) 2020 Martin Dobias
+*/
+
+#ifndef POSTGRESDRIVER_H
+#define POSTGRESDRIVER_H
+
+#include "driver.h"
+
+extern "C"
+{
+#include <libpq-fe.h>
+}
```

Comment:
this could be probably in cpp, right?

Reply:
That may cause problems - we're using `PGconn*` in the driver class, and `PGconn` is defined in libpq-fe.h as `typedef struct pg_conn PGconn;`